src/main/groovy/com/morpheusdata/nutanix/prismelement/plugin/backup/NutanixPrismElementBackupProvider.groovy

```diff
@@ -0,0 +1,15 @@
+package com.morpheusdata.nutanix.prismelement.plugin.backup
+
+import com.morpheusdata.core.MorpheusContext
+import com.morpheusdata.core.Plugin
+import com.morpheusdata.core.backup.MorpheusBackupProvider
+
+class NutanixPrismElementBackupProvider extends MorpheusBackupProvider {
+    NutanixPrismElementBackupProvider(Plugin plugin, MorpheusContext morpheusContext) {
+        super(plugin, morpheusContext)
+
+        NutanixPrismElementBackupTypeProvider nutanixPrismSnapshotProvider = new NutanixPrismElementBackupTypeProvider(plugin, morpheusContext)
+        plugin.pluginProviders.put(nutanixPrismSnapshotProvider.code, nutanixPrismSnapshotProvider)
+        addScopedProvider(nutanixPrismSnapshotProvider, "nutanix-prism-element-provision-provider", null)
```

Comment:
I believe this should be "nutanix" because of the overlapping cloud code. The plugin code is "nutanix-prism-element-provision-provider" but the provision type code is "nutanix". It should match the BackupIntegration seed.